src/main/java/com/eleks/academy/whoami/core/state/ProcessingQuestion.java

```diff
@@ -39,13 +43,39 @@ public void ask(String player, PlayerAction.Action question) {
 		throw new GameException("Not implemented");
 	}
 
-	;
+	public void answer(String player, String value) {
+		if (turn.getCurrentTurn().get(0).getValue() != null) {
+			if (isAnswerState()) {
+				turn.action(player, value);
+				answers.add(value);
+				if (answers.size() == players.size() - 1) {
+					turn.makeTurn(players.values().stream().toList(), collectAnswers());
+					answers.clear();
+					updatePlayersState(this.getCurrentTurn(), players);
+				}
+			} else {
```

Comment:
to not write such if-else cases you may do checks in the beggining of the method

Reply:
I don't really understand what to use
I try to use orElseThrow but this result is ignored, try to use if check in one line but it is incorrect.
![image](https://user-images.githubusercontent.com/49100972/180726670-03617639-c2c1-44a9-9abf-a6fe4e0cbe42.png)

Reply:
`if(!turn.isAnswerer(player)) throw new GameException("You can't answer")`